robomaker_simulation_msgs/README.md

```diff
@@ -1,162 +1,55 @@
-# AWS RoboMaker ROS Message Service
+# Robomaker ROS Service
```

Comment:
Did you intentionally clear the README?  It's now just headings.

Reply:
Yes, we need ROS2 examples added in.

Reply:
Update the commit description and the README to make it explicit that it is intentionally left blank and examples need to be added.